packer/linux/conf/docker/subgid

```diff
@@ -1,2 +1,2 @@
-buildkite-agent:1001:1
+buildkite-agent:993:1
```

Comment:
The docker rpm installed from the repos automatically creates a `docker` group with this gid.

Reply:
My spidey senses tingle a bit with this change.
I vaguely remember some fragile things hanging off these `subgid` mappings.
But, it's probably fine.
As long as we test it with a rootless container? Maybe via Buildkite's docker plugin or something?